### PR TITLE
Release v0.5.2

### PR DESCRIPTION
**Pull request by sargun**

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -1,3 +1,11 @@
+2016-03-04: v0.5.2
+  Add EnforceRFC952 to sample config
+  Improve performance of record generation
```
Comment (jdef) [inline CHANGELOG]: ugh, just realized that I never got around to benchmarking this.

File: CHANGELOG [hunk 2]
```diff
@@ -1,3 +1,11 @@
...
+  Fix bug in DiscoveryInfo to use the same scheme for record creation
+    as normal records (#377)
+  Add enumeration (#402)
```
Comment (jdef) [inline CHANGELOG]: "Add enumeration API" ?

File: CHANGELOG [hunk 3]
```diff
@@ -1,3 +1,11 @@
...
+  Generate additional SRV records if port name is specified
```
Comment (jdef) [inline CHANGELOG]: ".. if DiscoveryInfo port name ..."

File: CHANGELOG (hunk 1 above)
Comment (sargun) [inline CHANGELOG]: Do you want to do that before we proceed? I primarily did this PR to get the build for testing. We can stall if you don't feel comfortable

Comment (jdef) [inline CHANGELOG]: I can try to tackle that next week. Want to assign me a ticket?

**Pull request merged**